docs/process_joined_data.js

```diff
@@ -60,17 +66,28 @@ function colorAndRate(
 function processJoinedData(
     dataPromise, scalingFactor, initialTogglesValuesPromise) {
   return Promise.all([dataPromise, initialTogglesValuesPromise])
-      .then((results) => {
-        const [featuresList, {
-          povertyThreshold,
-          damageThreshold,
-          povertyWeight,
-        }] = results;
+      .then(([
+              featuresList,
+              {
+                povertyThreshold,
+                damageThreshold,
+                povertyWeight,
+              },
+            ]) => {
+        const columnsFound = new Set(ALWAYS_PRESENT_KEYS);
         for (const feature of featuresList) {
           colorAndRate(
               feature, scalingFactor, povertyThreshold, damageThreshold,
               povertyWeight);
+          for (const key of Object.keys(feature.properties)) {
+            if (!columnsFound.has(key) && !KEY_BLACKLIST.has(key)) {
```

Comment:
does the set not already handle the !columnsFound.has(key) part of this logic?

Reply:
Done. The "not being able to use add instead of contains" throws me off.